setup.py

```diff
@@ -33,16 +33,16 @@ def find_version(*file_paths):
 
 requirements = [
     'certifi>=2019.3.9',
-    'urllib3',
+    'urllib3>=1.15',
     'pyjwt>=1.7.1',
-    'six',
+    'six>=1.10',
```

Comment:
Copied constraints from the setup.py that is generated by the openapi-generator.